examples/01_plotting/plot_atlas.py

```diff
@@ -11,22 +11,39 @@
 
 from nilearn import datasets
 
-dataset = datasets.fetch_atlas_harvard_oxford('cort-maxprob-thr25-2mm')
-atlas_filename = dataset.maps
+dataset_ho = datasets.fetch_atlas_harvard_oxford('cort-maxprob-thr25-2mm')
+dataset_ju = datasets.fetch_atlas_juelich('maxprob-thr0-1mm')
 
-print('Atlas ROIs are located at: %s' % atlas_filename)
+atlas_ho_filename = dataset_ho.filename
```

Comment:
There is no `filename` key in this case because symmetric_split=False.

```suggestion
atlas_ho_filename = dataset_ho.maps
```